pkg/kn/plugin/manager.go

```diff
@@ -125,6 +136,9 @@ func (manager *Manager) ListPluginsForCommandGroup(commandGroupParts []string) (
 
 	// Examine all files in possible plugin directories
 	hasSeen := make(map[string]bool)
+	for _, pl := range plugins {
+		hasSeen[pl.Name()] = true
```

Comment:
I can change this line to set to `false` instead of `true` and all tests still pass...

Reply:
fixed